text/0005-architecture-reviews.md

```diff
@@ -0,0 +1,163 @@
+* Competitive analysis
+* Opportunity cost
+
+The key outcome is senior management investing in the problem space and building consensus around a shared reality.
+
+A Design Review should focus on:
+
+* The solution space
+* Technical roadmaps
+* Milestones and timelines
+* Resource allocation
+
+The key outcome is middle management and individual engineers understanding the value proposition and the path to success.
+
+In [RACI](https://en.wikipedia.org/wiki/Responsibility_assignment_matrix) terms, the team is Responsible, the chair is Accountable, other senior engineers or managers are Consulted and various others in the organization are Informed.
+
+### Structure and Inputs
+
+The second two considerations shape how the review process actually happens.
+
+The chair and the team frame the question or questions at the start of the process to ensure agreement on what should be tested. The chair is accountable for answering the question or questions following the review process.
+
+In order to agree on the question(s), the chair and the team/technical lead or principals will generally have an initial meeting. This meeting should be as small as possible and is intended to:
+
+* Establish vocabulary
+* Help the team "argue the right level" for the review
+* Identify stakeholders and domain experts — those Consulted, per RACI — in the organization
+
+Subsequently, the team prepares a review packet detailing the problem space and their approach.
+
+The goal is to have the chair (representing the responsible decision makers) and the team establish some shared reality informally, and then to have the team document this and explain how their work addresses the decision maker’s question within that reality.
+
+The chair then evaluates and refines the team’s document with the assistance of domain experts and the identified stakeholders. Hopefully the consulted stakeholders concur with the framing of the problem and are aligned behind the architectural approach, but perhaps they are not. The chair collects such concerns and presents them. This should be a collaborative process whereby the architectural review functions to document a problem and a proposed solution, in which case the chair can solicit response from the team.
+
+### Process and Conduct
+
+Involving the team and the chair informally and subsequently sharing ownership of a document is designed to seed a collaborative process. A particular benefit is avoiding drift between the proposal and the focus of the review.
+
+An Architecture Review process can be formally mandated. This may work in environments with a large degree of formality and process. This isn't how Mozilla works. For an Architecture Review process to be successful at Mozilla it must be beneficial to the team undergoing review.
+
+The primary informal output of a review should be learning. It is expected that authoritarian go/no-go decisions will fall out additional learning rather than being imposed in the form of a question.
```

Comment:
The intent was to say - Rather than asking "_Should we invade Russia in the winter: Yes/No_", and then get into an entrenched, heated debate, we instead ask "_Evaluate the plan to invade Russia in the winter_" and then learn about history and geography to jointly decide that there are better ideas.

I've written:

> The primary informal output of a review should be learning. It is expected that go/no-go decisions will be the result of additional learning rather than the team seeing a 'no-go' as something that needs fighting. Imposed decisions may be required, but learning is a preferred outcome.

Also, nit (s/out/fall out/): Fixed.